libmscore/hairpin.cpp

```diff
@@ -173,13 +184,20 @@ void HairpinSegment::draw(QPainter* painter) const
       else
             color = hairpin()->curColor();
       QPen pen(color, point(hairpin()->lineWidth()), hairpin()->lineStyle());
+      if (hairpin()->lineStyle() == Qt::CustomDashLine) {
+            QVector<qreal> pattern;
+            pattern << 5.0 << 20.0;
```

Comment:
OK good. I checked the frames and we use the same construct. Btw, I just remembered that the pattern thing is very slow for very long lines. We had this problem with frames in continuous view when the vbox were still displayed. I hope it will be ok in this case.